packages/x-license-pro/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-license-pro",
-  "version": "6.9.0",
+  "version": "6.9.1",
```

Comment:
This package only had a babel minor and `@mui/utils` patch version bumps, and they are not locked to specific version.
Do you think that it warrants a release? 🤔

Reply:
Gotcha. 👌